backend/de.metas.business/src/main/java/de/metas/order/compensationGroup/GroupCreator.java

```diff
@@ -82,6 +94,8 @@ public Group createGroup(
 						.orderLineIds(lineIdsToGroup != null ? ImmutableSet.copyOf(lineIdsToGroup) : ImmutableSet.of())
 						.newGroupTemplate(groupTemplate)
 						.newContractConditionsId(contractConditionsId)
+						.qtyMultiplier(qty)
+						.groupingProductId(groupingProductId)
```

Comment:
what's the actual reason of passing groupingProductId down the line?

Reply:
in `de.metas.order.compensationGroup.OrderGroupRepository#createRegularLineFromTemplate` I'm only allowing printing for the groupingProductId